HIP-0013.md

```diff
@@ -0,0 +1,130 @@
+
+### Domains as Portable Identities
+By attaching profile data such as a name, avatar, and bio, domains can act as portable online identities. This "bring-your-own-identity" approach, where users control their own data, has been adopted by platforms like [BlueSky](https://atproto.com/specs/handle), [Thunderbolt](https://www.spaceship.com/thunderbolt), and [Farcaster](https://docs.farcaster.xyz/learn/architecture/ens-names), which all utilize domains over traditional usernames.
+
+Domains help overcome the scarcity of desirable namespace on social networks, allowing for more personalized and meaningful identifiers. For example, a proud Texan might choose `@richard.tx` over settling for `@richard_214`. Similarly, verified domains like `@nike.com` establish true authenticity, a clear advantage over fallible methods like blue checkmarks.
+
+### Domains for Wallet Naming
+In the same way that domains provide human-readable names for IP addresses, they can also map human-readable names to lengthy cryptocurrency addresses. Wallet naming makes it easy to send crypto to a friend or donate to a worthy cause. As crypto adoption grows, domains must evolve to support the payment rails of the future. Wallet naming continues to be the primary driver of adoption for blockchain naming systems like [ENS](https://docs.ens.domains/web/resolution).
+
+### Advantages of TXT Records
+Utilizing DNS offers practical benefits over blockchain-based systems: 
+* **Compatible** - Works with both Handshake and ICANN domains across different registrars and DNS providers.
+* **Economical** - No gas fees needed for data storage or ongoing management.
+* **Ease of Use** - Simple for both domain owners and developers. No specialized blockchain knowledge required.
+* **Security** - DNSSEC ensures data integrity, while proven DNS infrastructure guarantees speed & availability.
+
+### DNS: A Strategy for Adoption
+This specification is fully compatible with ICANN domains, which dramatically broadens the potential user base. By growing the identity use-case for DNS domains, we can position Handshake as the superior alternative – private, decentralized, and censor-resistant. By building tools and services around IDNS that support both HNS and DNS domains, we can reach a much wider audience beyond the crypto community.
+
+## TXT Specification
+
+IDNS TXT records begin with **`IDNS1`** (version 1), and use the following format containing a record type, key, and value: 
+
+<pre><code><b>IDNS1 &lt;type&gt;:&lt;key&gt;=&lt;value&gt;</b>
+</code></pre>
+
+IDNS TXT records can contain one or more records in key=value pairs, separated by spaces:
+<pre><code><b>IDNS1</b> profile:avatar=<ins>http://pic.com/me.png</ins> service:com.twitter=<ins>aoxborrow</ins>
+</code></pre>
+
+Records that contain spaces must be single quoted. Single quotes in the record can be escaped with a backslash:
```

Comment:
no good reason actually, should change it since single is kinda fugly, i was thinking double quotes might break RR's but any decent DNS code will escape quotes correctly

Reply:
changed to just say quoted, and showed examples of both single and double quoted